recipes/darr/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "darr" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 183886142de227c8179409b8c69a5dfec0c74d467f9aa78be8224070961cad1b
+
+build:
+  number: 0
+  skip: True  # [py<36]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - numpy >=1.12
+
+test:
+  imports:
+    - darr
+
+about:
+  home: https://github.com/gbeckers/darr
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'A Python science library for memory-mapped numeric arrays, based on a format that is self-explanatory and tool-independent.'
+  description: |
+    Darr is a Python science library that enables you to work efficiently with
+    disk-based numeric arrays without depending on tool-specific data formats.
+    This makes it easy to share your data with those who do not use Darr or even
+    Python. No exporting required and, as the data is saved in a self-explanatory
+    way, not much explanation required either. Tool-independent and easy access
+    to data is in line with good scientific practice as it promotes wide and
+    long-term availability, to others but also to yourself.
+  doc_url: http://darr.readthedocs.io/
```

Suggest using `https` as RTD supports it.

```suggestion
  doc_url: https://darr.readthedocs.io/
```